src/friendly-errors-plugin.js

```diff
@@ -98,6 +99,31 @@ class FriendlyErrorsWebpackPlugin {
   }
 }
 
+function extractErrorsFromStats(stats, type) {
+  if (isMultiStats(stats)) {
+    const errors = stats.stats
+      .reduce((errors, stats) => errors.concat(extractErrorsFromStats(stats, type)), []);
+    // Dedupe to avoid showing the same error many times when multiple
+    // compilers depend on the same module.
+    return uniqby(errors, error => error.message);
+  }
+  return stats.compilation[type];
```

Comment:
If we have multi-stats, flatten everything, otherwise get the errors as usual.
Then ensure unicity of messages anyway.

I fell that it would make the code a clearer and would take a very small perf hit.
What do you think?